charts/openshift-reporting/templates/report-queries/persistentvolumeclaim-usage.yaml

```diff
@@ -42,3 +87,97 @@ spec:
         "timestamp",
         dt
     FROM {| dataSourceTableName "persistentvolumeclaim-usage-bytes" |}
+
+---
+
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "persistentvolumeclaim-usage-with-phase-raw"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportDataSources:
```

Comment:
This list is incorrect. This query uses 2 reportGenerationQueries, and the names here don't match either.